packages/amplify-cli/src/__tests__/commands/uninstall.pkg.test.ts

```diff
@@ -60,6 +60,18 @@ describe('uninstall packaged CLI on mac / linux', () => {
       value: originalPlatform,
     });
   });
+
+  expect.addSnapshotSerializer({
+    test(val) {
```

Comment:
My understanding is that Jest will call `test()`, and if `test()` returns a match, then it will serialize the value using `print()`. In that case, I think `test()` should check if the value is a string (like it currently is), and `process.platform === 'win32'` (since we only have this problem on Windows).

Reply:
```    // override process.platform
    Object.defineProperty(process, 'platform', {
      value: 'linux',
    });
```
Lines 101-104 ,override the platform to be linux ,thus path.join() will return with "/" joins but as Windows path location has "\\" , it could have been resolved by turning all "\\" into "/" when process.platform is not taken into consideration.

if process.platform==="win32" comes into play then since its being override by linux, it will go under no replacement 

So the best option I think would be not evaluate based on platform and try a standard of using "/" for all path joins, as it would be not bothered by any overrides and using path.posix.join() all paths in snapshots would have "/" and any windows location "\\" would be turned to "/"

So if any overrides are performed, regardless of that it will return standard "/" join path locations

Reply:
Instead of checking `process.platform`, can you check `originalPlatform === 'win32'` since that is the real value of `process.platform`.